New RUM Performance Monitoring summary page (#26703)
DataDog/documentation: content/en/real_user_monitoring/_index.md

Access RUM by navigating to [**Digital Experience > Performance Summary**][1].

> Select an application from the top navigation, or follow the setup instructions for [browser][15] or [mobile][16] to add your first one.

Reviewer: your first one what? application?

{{< img src="real_user_monitoring/rum-performance-application-selector.png" alt="Select a RUM application" >}}

**Tip**: To open RUM from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `real user monitoring`.

> ## Performance monitoring summary

Reviewer 2: I like the content, but I feel we're again missing something that shows what the mobile version looks like. Could we put two screenshots side by side, one for web, one for mobile, please?

Note: to take a good screenshot of a mobile app for the Demo org, you might want to deselect `user` from the Session Type filter as almost all of the traffic actually comes from Synthetics.

Author: Done - thanks for the suggestion!

> {{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance monitoring summary page for a browser application." >}}

Reviewer: ```suggestion
{{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance Monitoring summary page for a browser application" >}}
```

> The [RUM Performance Monitoring summary][1] page, which provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:

Reviewer: ```suggestion
The [RUM Performance Monitoring summary][1] page provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:
```

> - **Focus on key metrics** by platform, such as the UI latency for web or mobile crashes

Reviewer 2: ```suggestion
- **Focus on key data points** by platform, such as the UI latency for web or mobile crashes
```

Those are currently event based. They will be metric based in the future with RUM Tail-Based, but I'd rather use a generic term

> - **Monitor application health** through familiar KPIs, such as Core Web Vitals for web apps or hang rate for iOS to assess app reliability

Reviewer: ```suggestion
- **Monitor application health** through familiar KPIs, such as Core Web Vitals for web apps or hang rate for iOS, to assess app reliability
```

- **Dive into investigations directly** from interactive widgets without leaving the page

> For **web apps**, use the search bar to filter data, identify slow pages, and follow the UI to the [/rum/optimization/inspect][17] page.

Reviewer: ```suggestion
For **web apps**, use the search bar to filter data, identify slow pages, and follow the UI to the [RUM Optimization Inspect][17] page.
```
